### ISSUE TITLE
AppDeployToolkitMain.ps1 causing an error

### ISSUE THREAD

**Issue by bahnjee** [PSAppDeployToolkit/PSAppDeployToolkit#430]
Ran into a problem with my latest PSADT deployment and couldn't find what I might have broken, so I re-downloaded the Toolkit (after fighting Windows Defender saying the zip has a virus...separate issue submitted).  Still getting same error.

Even without editing anything and just running the toolkit as-is, it throws this error:

`C:\Temp\Test\Deploy-Application.ps1 : Module [C:\Temp\Test\AppDeployToolkit\AppDeployToolkitMain.ps1] failed to load: `
`Exception calling "Parse" with "1" argument(s): "String was not recognized as a valid Boolean."`
`At C:\Temp\Test\Deploy-Application.ps1:95 char:81`
`+ ... yToolkitMain -DisableLogging } Else { . $moduleAppDeployToolkitMain }`
`+                                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~`
`+ CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException`
`    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Deploy-Application.ps1`

**Comment by luki1412** [comment]
I cannot replicate this error. Did you use the Toolkit folder from the zip file?

**Comment by bahnjee** [comment]
Yes, sir, I did.  However, this morning, I'm not able to replicate the problem either.

Just tried downloading the package again using Chrome, Edge, Edge Dev, Firefox, and even IE.  Firefox is the only browser that didn't block the file as virus-infected.  When I ran Deploy-Application.ps1 (without any edits at all), it ran just fine.

I can only surmise that last night, something in the zip was modified/broken by the false positive virus detection.  I apologize for the trouble.
